Makefile

```diff
@@ -33,6 +51,12 @@ code/run:
 test/unit:
 	@./scripts/ci/unit_test.sh
 
+.PHONY: test/e2e/prow
+test/e2e/prow: setup/prow cluster/prepare
```

Comment:
This isn't really prow specific as far as i can tell, we should just be able to copy what we do in the integreatly operator https://github.com/integr8ly/integreatly-operator/blob/master/Makefile#L88-L92 (make test/e2e)